tasks/test.py

```diff
@@ -43,10 +43,10 @@ def regression(ctx):
     ],
 )
 def ci(ctx):
-    """Run tests in CircleCI.
+    """Run tests not requiring access to external data.
 
-    Excludes regression tests that require access to data on NSIDC-specific
-    infrastructure.
+    Excludes e.g., regression tests that require access to data on
```

Comment:
Does this also exclude integration tests?  I think of those as involving routines that require routines from more than one repository.

Reply:
Yes, currently this only runs `mypy` and unit tests.